docs/sources/query/_index.md

```diff
@@ -0,0 +1,655 @@
+| User           | User for the Project (shown when User was previously selected) | Yes |
+| Project Number | Enter a specific Project Number to query for associated items | No |
+| Filter         | Add key value filters based on the fields for project items (shown if Project Number specified) | No |
+
+##### Sample queries
+Show all projects for the `grafana/grafana` repository:
+
+- Project Owner: `organization`
+- Organization: `grafana`
+
+Show all pull requests for the "Dashboards" project in the Grafana organization:
+
+- Project Owner: `organization`
+- Organization: `grafana`
+- Project Number: `202`
+- Filter: `type equal PULL_REQUEST`
+
+#### Response
+
+##### When no Project Number is specified
+
+| Name              | Description |
+|-------------------|-------------|
+| number            | The project number | 
+| title             | Title of the project | 
+| url               | URL for the project |
+| closed            | Whether the project has been closed: `true`/ `false` |
+| public            | Whether the project is public: `true` / `false` |
+| closed_at         | When the project was closed: YYYY-MM-DD HH:MM:SS |
+| updated_at        | When the project was last updated |
+| created_at        | When the project was created | 
+| short_description | The description of the project |
+
+{{< admonition type="note" >}}
+This query returns a maximum of 200 results.
+{{< /admonition >}}
+
+##### When a Project Number is specified
+{{< admonition type="note" >}}
+GitHub Projects allow for customization of default fields and custom fields to be added, 
+therefore the response can vary significantly between projects."
+{{< /admonition >}}
+
+| Name            | Description |
+|-----------------|-------------|
+| name            | Name of the project item (issue or pull request) |
+| id              | Unique identifier for the project item           |
+| type            | Type of the item (such as ISSUE, PULL_REQUEST)     |
+| status          | Status of the item (such as "In development", "Shipped") - this can be configured on the project |
+| labels          | Comma-separated list of labels assigned to the item |
+| assignees       | Comma-separated list of users assigned to the item |
+| reviewers       | Comma-separated list of reviewers (for pull requests) |
+| repository      | Name of the repository the item belongs to       |
+| milestone       | Milestone associated with the item      |
+| priority        | Priority value or label               |
+| archived        | Whether the item is archived: `true` / `false`   |
+| created_at      | When the item was created: YYYY-MM-DD HH:MM:SS   |
+| updated_at      | When the item was last updated: YYYY-MM-DD HH:MM:SS |
+| closed_at       | When the item was closed, if applicable: YYYY-MM-DD HH:MM:SS |
+| (custom fields) | Any custom defined fields will also be returned alongside their values |
+
```

Comment:
should we move both notes to the bottom?

```suggestion
{{< admonition type="note" >}}
This query returns a maximum of 200 results.
{{< /admonition >}}

{{< admonition type="note" >}}
GitHub Projects allow for customization of default fields and custom fields to be added, 
therefore the response can vary significantly between projects."
{{< /admonition >}}
```

personally think its looks cleaner/easier to read with this change, WDYT?

<img width="49%" alt="Screenshot 2025-05-20 at 16 45 41" src="https://github.com/user-attachments/assets/e1388e09-2199-4836-9901-bfac84fcb78e" />

<img width="49%" alt="Screenshot 2025-05-20 at 16 46 37" src="https://github.com/user-attachments/assets/f979635a-741e-4511-a823-a2051c6d924b" />

Reply:
Hmm, the note on the customization only applies for the response where a project number is specified, whereas the query limit is generic... 

How about moving query limit notes consistently higher up - i.e. before query options?